txircd/ircd.py

```diff
@@ -449,10 +555,11 @@ def generateISupportList(self):
 		isupport = self.isupport_tokens.copy()
 		statusSymbolOrder = "".join([self.channelStatuses[status][0] for status in self.channelStatusOrder])
 		isupport["CHANMODES"] = ",".join(["".join(modes) for modes in self.channelModes])
+		isupport["CHANNELEN"] = self.config.get("channel_name_length", 64)
+		isupport["NETWORK"] = self.config["network_name"]
```

Comment:
You removed a bunch of hardcoded values above, but I don't see them here, nor do I see any actions to pull them in.